libplatsupport/CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ elseif(${KernelArch} STREQUAL "riscv")
     list(APPEND irqchip_modules "-Wl,--undefined=riscv_plic_ptr")
 endif()
 
-if(KernelPlatformQEMUArmVirt)
+if(KernelPlatformQEMUArmVirt OR KernelPlatformQuartz64)
```

Comment:
I wonder if we should make this generic for any ARM platform then? And add a ltimer config flag that builds can set to control the behavior explicitly.